addons/website_sale/controllers/main.py

```diff
@@ -288,10 +288,11 @@ def shop(self, page=0, category=None, search='', min_price=0.0, max_price=0.0, p
         options = self._get_search_options(
             category=category,
             attrib_values=attrib_values,
-            pricelist=pricelist,
+            pricelist=pricelist,  # TODO edm: check after merge
             min_price=min_price,
             max_price=max_price,
             conversion_rate=conversion_rate,
+            display_currency=website.currency_id,  # TODO edm check if adding this line is necessary
```

Comment:
:duck:

Reply:
push* I just wanted to build the runbot before to start building it. And then, as I forseen... I forgot. >.<